templates/default/plugin.conf.erb

```diff
@@ -2,7 +2,15 @@
 # This file is generated by Chef
 # Do not edit, changes will be overwritten
 #
+<% if @loadplugin_options.empty? %>
 LoadPlugin "<%= @name %>"
+<% else %>
+<LoadPlugin "<%= @name %>">
+  <% @loadplugin_options.each_pair do |key, value| %>
+  <%= key %> <%= collectd_option(value) %>
```

Comment:
It encloses strings with double-quotes.
See https://github.com/hectcastro/chef-collectd/blob/develop/libraries/default.rb